spec.html

```diff
@@ -4488,53 +4488,71 @@ <h2>Syntax</h2>
-            </li>
-            <li>
-              The MV of <emu-grammar>StrUnsignedDecimalLiteral ::: `.` DecimalDigits ExponentPart</emu-grammar> is the MV of |DecimalDigits| times 10<sup>_e_ - _n_</sup>, where _n_ is the number of code points in |DecimalDigits| and _e_ is the MV of |ExponentPart|.
-            </li>
-            <li>
-              The MV of <emu-grammar>StrUnsignedDecimalLiteral ::: DecimalDigits ExponentPart</emu-grammar> is the MV of |DecimalDigits| times 10<sup>_e_</sup>, where _e_ is the MV of |ExponentPart|.
-            </li>
-          </ul>
-          <p>Once the exact MV for a String numeric literal has been determined, it is then rounded to a value of the Number type. If the MV is 0, then the rounded value is *+0*<sub>𝔽</sub> unless the first non white space code point in the String numeric literal is `-`, in which case the rounded value is *-0*<sub>𝔽</sub>. Otherwise, the rounded value must be the Number value for the MV (in the sense defined in <emu-xref href="#sec-ecmascript-language-types-number-type"></emu-xref>), unless the literal includes a |StrUnsignedDecimalLiteral| and the literal has more than 20 significant digits, in which case the Number value may be either the Number value for the MV of a literal produced by replacing each significant digit after the 20th with a 0 digit or the Number value for the MV of a literal produced by replacing each significant digit after the 20th with a 0 digit and then incrementing the literal at the 20th digit position. A digit is significant if it is not part of an |ExponentPart| and</p>
-          <ul>
-            <li>
-              it is not `0`; or
-            </li>
-            <li>
-              there is a non-zero digit to its left and there is a non-zero digit, not in the |ExponentPart|, to its right.
-            </li>
-          </ul>
+        <emu-clause id="sec-runtime-semantics-stringnumericvalue" aoid="StringNumericValue" oldids="sec-runtime-semantics-mv-s">
+          <h1>Runtime Semantics: StringNumericValue</h1>
+          <p>The conversion of a String to a Number value is similar overall to the determination of the Number value for a numeric literal (see <emu-xref href="#sec-literals-numeric-literals"></emu-xref>), but some of the details are different, so the process for converting a String numeric literal to a value of Number type is given here.</p>
+          <emu-grammar>StringNumericLiteral ::: StrWhiteSpace?</emu-grammar>
+          <emu-alg>
+            1. Return *+0*<sub>𝔽</sub>.
+          </emu-alg>
+          <emu-grammar>StringNumericLiteral ::: StrWhiteSpace? StrNumericLiteral StrWhiteSpace?</emu-grammar>
+          <emu-alg>
+            1. Return StringNumericValue of |StrNumericLiteral|.
+          </emu-alg>
+          <emu-grammar>StrNumericLiteral ::: NonDecimalIntegerLiteral</emu-grammar>
+          <emu-alg>
+            1. Return RoundStringMVResult(MV of |NonDecimalIntegerLiteral|).
```

Comment:
```suggestion
            1. Return 𝔽(MV of |NonDecimalIntegerLiteral|).
```
In the current spec, the 20-digit rounding only applies to *decimal* literals. If you apply it to non-decimal literals, then there are cases where the spec would allow two valid outcomes where it currently only allows one, which would be a normative change.